### PR TITLE
[develop] Change file paths to EPIC data locations and add fix_gsi var

### PR DESCRIPTION
**Pull request by EdwardSnyder-NOAA**
## DESCRIPTION OF CHANGES: 

Updated the data file paths to point from SRW account locations to EPIC role account locations, so that EPIC can have better control and management of the SRW App data files. 

Also added the FIXgsi var.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## TESTS CONDUCTED: 

- [x] hera.intel
- [x] orion.intel
- [x] cheyenne.intel
- [ ] cheyenne.gnu
- [x] gaea.intel
- [x] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [ ] fundamental test suite
- [x] comprehensive tests (specify *which* if a subset was used)
nco_grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
community_ensemble_2mems
grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v16
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_HRRR
MET_verification

## DOCUMENTATION:

Updated RunSRW and ContainerQuickstart rst files. The files listed paths to the older location and have been updated to point to the EPIC location.

## ISSUE: 

This PR addresses the lack of FIXgsi directory in the develop branch as described in issue #672 

## CHECKLIST

- [x] My code follows the style guidelines in the Contributor's Guide
- [x] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain).
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## LABELS (optional): 

A Code Manager needs to add the following labels to this PR: 
- [x] Work In Progress
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] release
- [ ] high priority
- [ ] run_ci
- [ ] run_we2e_fundamental_tests
- [ ] run_we2e_comprehensive_tests
- [ ] Needs Cheyenne test 
- [ ] Needs Jet test 
- [ ] Needs Hera test 
- [ ] Needs Orion test 
- [ ] help wanted

## CONTRIBUTORS (optional): 

@natalie-perlin for her help moving data, documenting, and testing the changes.

### CODE REVIEW

File: ush/machine/gaea.yaml [hunk 1]
```diff
@@ -18,16 +18,17 @@ platform:
...
+  FIXaer: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/fix/fix_aer
+  FIXgsi: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/fix/fix_gsi
+  FIXgsm: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/fix/fix_am
+  FIXlut: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/fix/fix_lut
+  FIXorg: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/fix/fix_orog
+  FIXsfc: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/fix/fix_sfc_climo
+  FIXshp: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/NaturalEarth
 data:
   ics_lbcs:
     FV3GFS: /lustre/f2/dev/Mark.Potts/EPIC/SRW/model_data/FV3GFS/${yyyymmdd}${hh}
```
Comment (natalie-perlin) [inline ush/machine/gaea.yaml]: This line may need to be changed, too, as it points to the old location.
Suggesting:

```
  FV3GFS:
    nemsio: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/input_model_data/FV3GFS/nemsio/${yyyymmdd}${hh}
    grib2: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/input_model_data/FV3GFS/grib2/${yyyymmdd}${hh}
```

File: ush/machine/noaacloud.yaml [hunk 2]
```diff
@@ -14,17 +14,18 @@ platform:
...
+  FIXgsi: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_gsi
+  FIXgsm: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_am
+  FIXlut: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_lut
+  FIXorg: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_orog
+  FIXsfc: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_sfc_climo
+  FIXshp: /contrib/EPIC/UFS_SRW_data/develop/NaturalEarth
+  FIXgsi: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_gsi
   EXTRN_MDL_DATA_STORES: aws nomads
 data:
   ics_lbcs:
```
Comment (natalie-perlin) [inline ush/machine/noaacloud.yaml:31]: Suggest to modify the subsequent line to:

  ```
  FV3GFS:
       nemsio: /contrib/EPIC/UFS_SRW_data/develop/input_model_data/FV3GFS/nemsio/${yyyymmdd}${hh}
       grib2: /contrib/EPIC/UFS_SRW_data/develop/input_model_data/FV3GFS/grib2/${yyyymmdd}${hh}
```

**Review by natalie-perlin — COMMENTED** [review]
It would be good to get any comments or suggesions on how to deal with the old locations specified under "data" section of the machine.yaml file

File: ush/machine/noaacloud.yaml (hunk 2 above)
Comment (EdwardSnyder-NOAA) [reply ush/machine/noaacloud.yaml:31]: done.

**Comment by EdwardSnyder-NOAA** [comment]
@natalie-perlin - the paths found under the "data" section are the default locations of the data files for each platform. Setting this variable to true: `USE_USER_STAGED_EXTRN_FILES` will trigger the generation script to look at our EPIC location via this variable:
`EXTRN_MDL_SOURCE_BASEDIR_*`. 

I did change the two "data" paths that were pointing to older locations to our EPIC ones. I don't think any further modifications are needed.

**Review by natalie-perlin — APPROVED** [review]
@EdwardSnyder-NOAA - thank you for taking care of the new FIXgsi data on the platforms!

**Comment by gsketefian** [comment]
@EdwardSnyder-NOAA FYI In my PR #683 I added new data on Hera.  Details:
1) Added the directory `/scratch2/BMC/det/UFS_SRW_App/develop/output_data/fcst_det/RRFS_CONUS_25km/rrfs.20190615`.   This just contains some symlinks to enable running the workflow in nco mode.  The symlinks are in the subdirectory `00` and point to files in `/scratch2/BMC/det/UFS_SRW_App/develop/output_data/fcst_det/RRFS_CONUS_25km/2019061500/postprd`.
2) Added the directory `/scratch2/BMC/det/UFS_SRW_App/develop/output_data/fcst_ens`.  This is to enable ensemble verification from staged data.  There are actual data files (in the subdirectories `RRFS_CONUS_25km/2019061500/mem[001|002]/postprd`) for running in community mode and symlinks to those files (in the subdirectory `rrfs.20190615/00`) for running in nco mode.

So these need to be copied over to the EPIC locations and the variable `TEST_VX_FCST_INPUT_BASEDIR` in `ush/machines/hera.yaml` updated accordingly (please also see my version of this file since the variable now contains some jinja logic that needs to be retained).  Thanks!

**Review by MichaelLueken — APPROVED** [review]
@EdwardSnyder-NOAA These changes look good to me!  I will go ahead and approve these changes now, but as @gsketefian noted, please make sure that the new data on Hera has been added to the EPIC maintained data locations.

**Comment by EdwardSnyder-NOAA** [comment]
@gsketefian & @MichaelLueken - I've updated the data in our EPIC data locations as described in Gerard's comment and updated the jinja logic in the hera.yaml.

**Comment by gsketefian** [comment]
@EdwardSnyder-NOAA Your change looks good to me.  Thanks.

**Review by christinaholtNOAA — APPROVED** [review]
LGTM

**Comment by MichaelLueken** [comment]
@EdwardSnyder-NOAA I have manually reviewed the logs in the Jenkins workspaces on the various machines to make sure that all tests successfully passed (since Jenkins is currently down due to migration to a new site).  All Jenkins tests have passed.  Since you have the necessary approvals, I will go ahead and merge this work now.  I have also removed your issue #672 from being closed, as it looks like you will need to add supported grids for RRFS.  Thanks!

**Pull request merged**